app/experimenter/experiments/migrations/0149_auto_20210108_1427.py

```diff
@@ -0,0 +1,139 @@
+        ("experiments", "0148_auto_20201216_1815"),
+    ]
+
+    operations = [
+        migrations.RunPython(set_owner),
+        migrations.AlterField(
+            model_name="nimbusexperiment",
+            name="application",
+            field=models.CharField(
+                choices=[("firefox-desktop", "Desktop"), ("fenix", "Fenix")],
+                default="firefox-desktop",
+                max_length=255,
+            ),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name="nimbusexperiment",
+            name="channel",
+            field=models.CharField(
+                choices=[
+                    ("", "No Channel"),
+                    ("beta", "Desktop Beta"),
+                    ("nightly", "Desktop Nightly"),
+                    ("release", "Desktop Release"),
+                    ("default", "Desktop Unbranded"),
+                    ("org.mozilla.firefox.beta", "Fenix Beta"),
+                    ("org.mozilla.fenix", "Fenix Nightly"),
+                    ("org.mozilla.firefox", "Fenix Release"),
+                ],
+                default="",
+                max_length=255,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="nimbusexperiment",
+            name="firefox_min_version",
+            field=models.CharField(
+                choices=[
+                    ("", "No Version"),
+                    ("80.!", "Firefox 80"),
+                    ("81.!", "Firefox 81"),
+                    ("82.!", "Firefox 82"),
+                    ("83.!", "Firefox 83"),
+                    ("84.!", "Firefox 84"),
+                    ("85.!", "Firefox 85"),
+                    ("86.!", "Firefox 86"),
+                    ("87.!", "Firefox 87"),
+                    ("88.!", "Firefox 88"),
+                    ("89.!", "Firefox 89"),
+                    ("90.!", "Firefox 90"),
+                    ("91.!", "Firefox 91"),
+                    ("92.!", "Firefox 92"),
+                    ("93.!", "Firefox 93"),
+                    ("94.!", "Firefox 94"),
+                    ("95.!", "Firefox 95"),
+                    ("96.!", "Firefox 96"),
+                    ("97.!", "Firefox 97"),
+                    ("98.!", "Firefox 98"),
+                    ("99.!", "Firefox 99"),
+                    ("100.!", "Firefox 100"),
+                ],
+                default="",
+                max_length=255,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="nimbusexperiment",
+            name="hypothesis",
+            field=models.TextField(
+                default="If we <do this/build this/create this change in the experiment> for <these users>, then we will see <this outcome>.\nWe believe this because we have observed <this> via <data source, UR, survey>.\n\nOptional - We believe this outcome will <describe impact> on <core metric>\n    "
```

Comment:
I added trimming in the client to deal with it, but is there a particular reason we have the extra newline and spaces at the end of this?

Reply:
Oh good question probably not 🤷‍♀️